content/en/docs/concepts/overview/working-with-objects/labels.md

```diff
@@ -92,6 +92,10 @@ them.
 For some API types, such as ReplicaSets, the label selectors of two instances must not overlap within a namespace, or the controller can see that as conflicting instructions and fail to determine how many replicas should be present.
 {{< /note >}}
 
+{{< caution >}} 
```

Comment:
`{{< caution >}}` calls out things that cluster operators need to know so that they don't break their cluster. For example a security warning.

I can see that you _could_ misunderstand labels and misconfigure a cluster but the AND behaviour seems the safer one for many operations. Bearing that in mind I'm not sure that `{{< caution >}}` is the right short code here.

Reply:
Ok, that makes sense, sorry I saw it in the contributor guide as well so I wasn't aware caution was only used for that purpose in docs. What do you think instead then? Does it warrant a `note`? I feel like `note` is typically extra information so people may be tempted to skip over it, this is more signaling a common/potential pitfall. Is there anything equivalent to `caution` except for application developers?

Reply:
Maybe caution is OK. You've convinced me!